qa/tasks/mgr/dashboard/test_cephfs.py

```diff
@@ -71,3 +71,68 @@ def test_cephfs_list(self):
         self.assertIn('mdsmap', cephfs)
         self.assertIsNotNone(cephfs['id'])
         self.assertIsNotNone(cephfs['mdsmap'])
+
+    def test_ls_mk_rm_dir(self):
+        data = self._get("/api/cephfs/1/ls_dir", params={'path': '/'})
```

Comment:
I'm getting `CephFS id 1 not found` error when testing.
`fs_id` should not be hard-coded, it can be retrieved with `self.fs.get_namespace_id()`